### PR TITLE
Cleanups for tests in prep for switch to pootle fs

### PR DESCRIPTION
**Pull request by phlax**
- Setup fs config for all projects in pytest env

### CODE REVIEW

File: tests/models/revision.py [hunk 1]
```diff
@@ -45,18 +41,17 @@ def test_max_revision(revision, project0_nongnu, store0):
 def test_revision_incr(store0):
     """Tests revision is incremented when units change."""
     previous_revision = Revision.get()
-    db_unit = _update_translation(store0, 0, {'target': [u'Fleisch']},
-                                  sync=False)
-
+    db_unit = store0.units.exclude(target_f="").first()
```
Comment (unho) [inline tests/models/revision.py]: Why exclude units with blank target?

**Comment by unho** [comment]
lgtm once Travis passes.

File: tests/models/revision.py (hunk 1 above)
Comment (phlax) [inline tests/models/revision.py]: not sure it matters, but it was to get a translated unit

Comment (unho) [inline tests/models/revision.py]: I don't think it matters, just curious why the qs was being filtered at all. I am happy leaving this as is provided Travis passes.

**Pull request merged**